runtime/doc/news.txt

```diff
@@ -269,6 +269,10 @@ These existing features changed their behavior.
 • |vim.on_key()| callbacks won't be invoked recursively when a callback itself
   consumes input.
 
+• |q| in man pages now uses |CTRL-W_q| instead of |CTRL-W_c| to close the
```

Comment:
No point in linking to `|q|`, its doc is irrelevant/misleading in this context.
```suggestion
• "q" in man pages now uses |CTRL-W_q| instead of |CTRL-W_c| to close the
```